tests/playbooks/e2e.yaml

```diff
@@ -93,7 +94,29 @@
         name: '{{ vm_name }}'
         namespace: default
 
-    - name: Incrase the resources of VM
+    - name: Wait for VMI status to running
```

Comment:
You don't have to. The `kubevirt_vm` with `status=running` will wait for you,

Reply:
@machacekondra okay, should I remove it?

Reply:
Yes please. If it's removed it will then properly test if the module is waiting.